docs/appendix.rst

```diff
@@ -19,79 +77,105 @@ The best way to generate an SSH key in Windows is to use PuTTY_.
 .. _PuTTY: https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html
 
 1. Download and install PuTTY_.
-#. Under the `PuTTY` group in the windows start menu, run the `PuTTYgen` application.
-#. Click the ``Generate`` button, and follow the on-screen instructions to generate a random SSH key pair.
+#. Under the **PuTTY** group in the windows start menu, run the **PuTTYgen**
+   application.
+#. Click the **Generate** button, and follow the on-screen instructions to
+   generate a random SSH key pair.
 
 Exporting the SSH *private key*:
 
-1. Click the ``Conversions`` drop-down menu, and select ``Export OpenSSH key``. Leave the ``Key passphrase`` blank.
-#. Choose a save location for the SSH key (e.g., the install location of ``nengo-fpga``).
+1. Click the **Conversions** drop-down menu, and select **Export OpenSSH key**.
+   Leave the **Key passphrase** blank.
+#. Choose a save location for the SSH key (e.g., the install location of
+   `nengo-fpga`).
 
 Exporting the SSH *public key*:
 
-1. Keep ``PuTTYgen`` open. Do not regenerate the SSH key pair.
-#. SSH into the FPGA board using a username and password combination. The username used will have the SSH keys generated with these steps associated with it.
-#. Edit the ``~/.ssh/authorized_keys`` file with a text editor (e.g., ``nano``, ``vi``)
-#. Copy the *public key* text from the "Public key for pasting into OpenSSH authorized_keys file" area of ``PuTTYgen`` and paste it at the end of the ``authorized_keys`` file.
+1. Keep PuTTYgen open. Do not regenerate the SSH key pair.
+#. SSH into the FPGA board using a username and password combination.
+   The username used will have the SSH keys generated with these steps
+   associated with it.
+#. Edit the `~/.ssh/authorized_keys` file with a text editor
+   (e.g., ``nano``, ``vi``).
+#. Copy the *public key* text from the **Public key for pasting into OpenSSH
+   authorized_keys file** area of PuTTYgen and paste it at the end of the
+   `authorized_keys` file.
 
 
 Linux and Mac
 -------------
 
-1. Open a terminal. If SSH is not installed, install it with ``sudo apt-get install ssh``.
+1. On your computer, open a terminal. If SSH has not installed been installed
```

Comment:
This list is spaced out because there is a command on it's own line. Force a compact list by adding `.. rst-class:: compact` right before the list.